bodhi/tests/server/services/test_updates.py

```diff
@@ -291,6 +291,36 @@ def test_new_rpm_update_koji_error(self, publish, *args):
         self.assertEquals(up['errors'][0]['description'],
                           "Koji error getting build: bodhi-2.0.0-2.fc17")
 
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_koji_config_url(self, publish, *args):
+        """
+        Test html rendering of default build link
+        """
+        self.app.app.registry.settings['koji_web_url'] = u'https://koji.fedoraproject.org/koji/'
+        nvr = u'bodhi-2.0.0-2.fc17'
+        resp = self.app.post_json('/updates/', self.get_update(nvr))
+
+        resp = self.app.get('/updates/%s' % nvr, headers={'Accept': 'text/html'})
+
+        self.assertRegexpMatches(str(resp), ('https://koji.fedoraproject.org/koji'
+                                             '/search\?terms=.*\&amp;type=build\&amp;match=glob'))
+
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_koji_config_mock_url_without_trailing_slash(self, publish, *args):
+        """
+        Test html rendering of build link using a mock config variable 'koji_web_url'
+        without a trailing slash in it
+        """
+        self.app.app.registry.settings['koji_web_url'] = u'https://host.org'
```

Comment:
Let's add one more test with the setting set like this:

```
https://koji.fedoraproject.org/koji
```

If the user types something like the above, make sure the right thing happens.